src/webui/components/Package/index.js

```diff
@@ -1,62 +1,90 @@
+/**
+ * @prettier
+ * @flow
+ */
+
 import React from 'react';
-import PropTypes from 'prop-types';
-import Chip from '@material-ui/core/Chip';
-import { Link } from 'react-router-dom';
+import type { Element } from 'react';
+import { margin } from 'polished';
+
+import Tag from '../Tag';
+import { formatDate, formatDateDistance } from '../../utils/package';
 
-import { formatDateDistance } from '../../utils/package';
+import { IProps } from './types';
+import { Wrapper, Header, A, Name, Version, Overview, OverviewItem, Icon, Text, Details, Avatar, Author, Field, Content, Footer } from './styles';
 
-import classes from './package.scss';
+const getInitialsName = (name: string) =>
+  name
+    .split(' ')
+    .reduce((accumulator, currentValue) => accumulator.charAt(0) + currentValue.charAt(0), '')
+    .toUpperCase();
 
-const Package = ({ name, version, author, description, license, time, keywords }) => {
-  return (<section className={classes.package}>
-    <Link to={`detail/${name}`}>
-      <div className={classes.header}>
-        <div className={classes.title}>
-          <h1>
-            {name} <Chip label={`v${version}`} />
-          </h1>
-        </div>
-        <div role="author" className={classes.author}>
-          {author ? `By: ${author}` : ''}
-        </div>
-      </div>
-      <div className={classes.footer}>
-        <p className={classes.description}>
-          {description}
-        </p>
-      </div>
-      <div className={classes.tags}>
-        {keywords && keywords.map((keyword, index) => (
-          <Chip
-            key={index}
-            label={keyword}
-            className={classes.tag}
-          />
+const Package = ({
+  name: label,
+  version,
+  time,
+  author: { name = 'Anonymous', avatar = '', email = '' },
```

Comment:
side note: In `npm` an author can be a string or object.

Reply:
I didn't know...ok